src/main/java/org/zone/commands/structure/region/flags/messages/leaving/ZoneFlagLeavingSetMessageCommand.java

```diff
@@ -54,10 +68,14 @@
     public @NotNull CommandResult run(CommandContext commandContext, String... args) {
         Zone zone = commandContext.getArgument(this, ZONE);
         Component message = commandContext.getArgument(this, MESSAGE);
+        MessageDisplayType<?> displayMode = commandContext.getArgument(this, DISPLAY_MODE);
+        if (message == Messages.getEnterLeavingMessage()) {
```

Comment:
This can never be true

Reply:
but IJ doesn't complain? and yeah changing it to .equals() would be same right?

Reply:
== is memory equals, not comparison equals so unless it has the same memory address then it wont be true. 

Ij doesnt complain as IJ isnt smart enough to known Messages are created every time you call it (and therefore can never be the same instance (same memory address))

Reply:
sad